devdoc/docs/ContributeDocs.md

```diff
@@ -0,0 +1,394 @@
+    such, this does not hold up well over time.
+
+    Do not refer to how things were done in the past. For example, "in
+    the past we had to do X-Y-Z steps and now we no longer have to. This
+    ‘new feature’ can do it in one step, Z". This only adds potential
+    confusion. Just instruct on exactly what needs to be done now, using
+    words efficiently as possible, so for this example just say “perform
+    step Z”
+
+## Terms 
+
+There are many terms specific to HPCC Systems® and the HPCC Systems
+platform. Use the following style guide for word usage and
+capitalization guidelines to use when referring to system components or
+other HPCC-specific entities. Maintain consistent usage throughout all
+docs.
+
+### HPCC Systems® 
+
+Officially and legally the platform's name is HPCC Systems® and it is a
+registered trademark.
+
+You should always refer to the platform as HPCC Systems® and the
+registration mark ® should appear in the first and most prominent
+mention of the name HPCC Systems®.
+
+While it is acceptable to use the ® anywhere in a document, it is
+required to be used in the first and most prominent mention - so the
+average reader will be aware. Any usage after that first and most
+prominent is optional.
+
+**Components and Tools**
+
+-   HPCC Systems Platform
+
+-   Dali
+
+-   Sasha
+
+-   Thor
+
+-   hThor
+
+-   ROXIE
+
+-   DFU Server (Distributed File Utility Server)
+
+-   ESP Server (Enterprise Server Platform)
+
+-   ESP Services
+
+-   WsECL
+
+-   ECL Watch
+
+-   ECL Server
+
+-   ECLCC Server
+
+-   ECL Agent
+
+-   ECL IDE
+
+-   ECL Plug-in for Eclipse
+
+-   ECL Playground
+
+-   LDAP
+
+-   dafilesrv
+
+-   VS Code (no hyphen)
+
+-   ECL Language Extension for VS Code
+
+-   Configuration Manager (not ConfigMgr or ConfigManager)
+
+**Note:** when referring to the startup command, use configmgr (always
+lowercase)
+
+### Other Terms 
+
+-   HPCCSystems.com or [http://HPCCSystems.com](http://hpccsystems.com/)
+    (do not include the www portion)
+
+-   ECL (Enterprise Control Language)
+
+-   ECL command-line interface
+
+**Note:** when referring to the ECL command-line tool command, ecl is
+lowercase
+
+-   DFU Plus command-line interface
+
+<!-- -->
+
+-   cloud-native (always hyphenated when used as an adjective)
+
+<!-- -->
+
+-   DFU Workunits
+
+-   ECL Workunits
+
+-   Workunit
+
+-   WUID
+
+-   HPCC Systems®
+
+-   multi-node
+
+-   Superfiles, subfiles
+
+-   package map
+
+-   The **username** is the (usually unique) thing you type in with your
+    password, for example: bobsmith66.
+
+-   The **user name** is the name of the user, the user's real-life
+    name, for example: Bob Smith.
+
+###  Common Documentation terms 
+
+Use the following conventions for these commonly used terms:
+
+-   right-click
+
+-   double-click
+
+-   drag-and-drop
+
+-   click-and-drag
+
+-   plug-in
+
+-   drop list
+
+-   bare metal (no hyphen)
+
+-   blue-green (not Blue/Green)
```

Comment:
picky: (since this document is being picky) what is the reason for this?  It is not a mixture of the two it is two alternatives.

Reply:
I chose blue-green over blue/green because it wss the most common usage I found and I had to pick one for the sake of consistency. If you prefer blue/green I am ok with either. 
In fact, now that you had me take a closer look, I think blue/green **is** better.